schema.py

```diff
@@ -13,9 +13,7 @@ def __init__(self, autos, errors):
     @property
     def code(self):
         def uniq(seq):
-            seen = set()
-            seen_add = seen.add
-            return [x for x in seq if x not in seen and not seen_add(x)]
+            return list(set(seq))
```

Comment:
You're right, I'm getting errors going this so I'll hold off. I'll add a comment saying order will be lost instead, thank you.